src/sdk/model/user/user-entity-history-subscription-delete-request.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  UserEntityHistorySubscriptionDeleteRequestJson
+} from './__json__/user-entity-history-subscription-delete-request-json';
+
+/**
+ * User Entity History Subscription Delete Request
+ */
+/* istanbul ignore next: autogenerated */
+export class UserEntityHistorySubscriptionDeleteRequest {
+
+  constructor(private _json: UserEntityHistorySubscriptionDeleteRequestJson) {
```

Comment:
is this a good request class for having another constructor that takes `entityUuid` as a param?